options/options.go

```diff
@@ -2215,3 +2250,32 @@ func newSrcInfo(path []int32, children sourceinfo.OptionChildrenSourceInfo) sour
 		Children: children,
 	}
 }
+
+type lenientHandler interpreter
+
+func (h *lenientHandler) HandleErrorf(span ast.SourceSpan, msg string, args ...interface{}) error {
+	if h.lenienceEnabled {
+		h.lenientErrReported = true
+		return nil
+	}
+	return reporter.Error(span, fmt.Errorf(msg, args...))
```

Comment:
Should the reporter here not delegate to the provided reporter?

Reply:
In lenient mode, there is no provided reporter.